integration/v7/isolated/scale_command_test.go

```diff
@@ -300,6 +301,10 @@ var _ = Describe("scale command", func() {
 						buffer := NewBuffer()
 						_, err := buffer.Write([]byte("y\n"))
 						Expect(err).ToNot(HaveOccurred())
+
+						//Delay to reduce flakiness
+						time.Sleep(3 * time.Second)
```

Comment:
Per conversation with @gururajsh in order to address issues with flakey tests, we will be taking this as a tech debt.